test/chart.test.ts

```diff
@@ -15,6 +15,23 @@ test('empty stack', () => {
   expect(Testing.synth(chart)).toMatchSnapshot();
 });
 
+test('disabling resource name hashes at chart level', () => {
```

Comment:
Can you add a happy case scenario here too? Which would be that automatic hashes are working by default.